docs/shard.yml.schema.json

```diff
@@ -0,0 +1,255 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "shard.yml",
+  "description": "Metadata for projects managed by shards",
+  "type": "object",
+  "properties": {
+    "name": {
+      "type": "string",
+      "title": "project name",
```

Comment:
I believe the titles should be titlem-cased.

Reply:
I was going off of schema store's recommendations which (I interpreted as) titles being lowercase (https://github.com/SchemaStore/schemastore/blob/master/CONTRIBUTING.md#best-practices). I'm fine either way though.

Reply:
I don't know about title case in general but I would say that proper nouns (`GitHub`) as well as abbreviations (`URL`) should likely use upper case.

Reply:
The title is used by vscode to give the title for the pop-up, without it it just uses the file name:
![image](https://github.com/crystal-lang/shards/assets/32797552/d6835560-921a-495d-9db0-0d6c68c132f1)
![image](https://github.com/crystal-lang/shards/assets/32797552/d1160978-9c08-4755-a12f-dd2aa5f5b517)

Reply:
There are also instances where the title isn't necessarily the same as the key name
![image](https://github.com/crystal-lang/shards/assets/32797552/6c538177-6fdf-4006-b99c-8e19f40e46eb)
![image](https://github.com/crystal-lang/shards/assets/32797552/1beba79e-072c-4ffd-bc5e-507c10f9bef9)

Reply:
Sigh.